ld/scripttempl/elfarc.sc

```diff
@@ -262,6 +262,11 @@ cat <<EOF
   } =${NOP-0}
 
   ${DATA_PLT-${BSS_PLT-${PLT}}}
+  .jlitab       ${RELOCATING-0} :
```

Comment:
Should this be removed as well as the Linux linker changes since JLI is only supported by ARCv2 processors?